warp10/src/main/java/io/warp10/continuum/store/Store.java

```diff
@@ -360,6 +362,16 @@ public void run() {
             if (null != properties.getProperty(io.warp10.continuum.Configuration.STORE_KAFKA_DATA_CONSUMER_CLIENTID)) {
               props.setProperty("client.id", properties.getProperty(io.warp10.continuum.Configuration.STORE_KAFKA_DATA_CONSUMER_CLIENTID));
             }
+            if (null != properties.getProperty(io.warp10.continuum.Configuration.STORE_KAFKA_DATA_CONSUMERID_INFIX)) {
+              // If a consumerId inflix is provided, the consumerId is built the same way than inside the Kafka ConsumerConnector with the inflix part being suffixed to the hostname
+              UUID uuid = UUID.randomUUID();
+              String consumerUuid = String.format("%s_%s_%d-%s",
+                      InetAddress.getLocalHost().getHostName(),
+                      properties.getProperty(io.warp10.continuum.Configuration.STORE_KAFKA_DATA_CONSUMERID_INFIX),
+                      System.currentTimeMillis(),
+                      Long.toHexString(uuid.getMostSignificantBits()).substring(0,8));
```

Comment:
I kept it ISO with the Kafka ConsumerConnector implementation which implement it this way.